Documentation/gettingstarted/istio.rst

```diff
@@ -35,12 +35,12 @@ proxy, but that will only work if mTLS is not used.
 
 .. note::
 
-   If Cilium is deployed with the kube-proxy replacement, you need to set
-   ``bpf-lb-sock-hostns-only: true`` in the deployment yaml
-   directly or via ``hostServices.hostNamespaceOnly`` option with Helm.
-   Without this option, when Cilium does service resolution via
-   socket load balancing, Istio sidecar will be bypassed, resulting
-   in loss of Istio features including encryption and telemetry.
+   By default Cilium is deployed with the kube-proxy replacement so
```

Comment:
Unless I'm missing some more specific context, that's not true. Cilium is deployed with KPR disabled in both Helm and cilium-cli by default, since v1.10.

Reply:
I got confused about this, thanks for noting this. Fixed :-)